chart/k8gb/templates/service-delete-hook/job.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: hook-service-delete
```

Comment:
Is this hook going to be executed on every upgrade?

Reply:
Yes

Reply:
@k0da it will mean coredns service recreation every time? It is quite disruptive especially in case of NLB association / recreation